test/index.js

```diff
@@ -120,6 +120,20 @@ describe('sasslint-loader', function () {
     });
   });
 
+  it('should work with CSS, SCSS, SASS and LESS files by default', function (done) {
+    var config = {
+      context: './test/fixtures/test8',
+      entry: './index'
+    };
+
+    pack(assign({}, baseConfig, config), function (err, stats) {
```

Comment:
This method's been updated to return a promise which makes writing these tests a bit easier :) (#27)

```js
return pack(/* config */)
  .then(function (stats) {
    //asserts on stats
  });
```